istio/setup.cfg

```diff
@@ -0,0 +1,10 @@
+[aliases]
+test = pytest
+
+[tool:pytest]
+testpaths = ./tests
+
+[flake8]
+exclude = .eggs,.tox,build
+max-line-length = 700
+ignore = E128,E203,E226,E231,E241,E251,E261,E265,E302,E303,E731,W503,E266
```

Comment:
we should try to remove this

Reply:
I copied the list from the other setup.cfg files